public/app/plugins/panel/heatmap/module.tsx

```diff
@@ -52,7 +52,15 @@ export const plugin = new PanelPlugin<Options, GraphFieldConfig>(HeatmapPanel)
         // NOTE: this feels like overkill/expensive just to assert if we have an ordinal y
         // can probably simplify without doing full dataprep
         const palette = quantizeScheme(opts.color, config.theme2);
-        const v = prepareHeatmapData(context.data, undefined, opts, palette, config.theme2);
+        const v = prepareHeatmapData(
+          context.data,
+          undefined,
+          opts,
+          palette,
+          config.theme2,
+          undefined,
+          context.replaceVariables
+        );
         isOrdinalY = readHeatmapRowsCustomMeta(v.heatmap).yOrdinalDisplay != null;
       } catch {}
```

Comment:
What's this `catch` block for?

Reply:
My guess is one of the three functions in the try block might throw an error, but we don't care about the error. The only thing we care about is to set `isOrdinalyY`